tests/migration_test.go

```diff
@@ -33,6 +33,10 @@ import (
 	"sync"
 	"time"
 
+	migrationsv1 "kubevirt.io/api/migrations/v1alpha1"
+
+	k6tpointer "kubevirt.io/kubevirt/pkg/pointer"
```

Comment:
duplicate import, see `kvpointer`

Reply:
Thanks!
We need a linter here :)

Reply:
Seems it distinguishes between import groups by a newline separator.